docs/make.jl

```diff
@@ -3,6 +3,7 @@ using Attractors
 using Attractors.DynamicalSystemsBase
 using Attractors.StateSpaceSets
 using PredefinedDynamicalSystems
+using OrdinaryDiffEq:Vern9
```

Comment:
This does not need to be in the make.jl file. For dependencies used within the `@example` blocks in the docs, add a dependency in `docs/Project.toml`.